File: group 25/level 024/homework/task 03/homework3.py
def get_score(prompt):
    while True:
        try:
            score = float(input(prompt))
            if 0 <= score <= 100:
                return score
            else:
                print("ქულა უნდა იყოს 0-დან 100-მდე.")
        except ValueError:
            print("გთხოვთ შეიტანოთ ვალიდური რიცხვი.")

# შეიტანეთ ქულები
midterm_score = get_score("შეიტანეთ შუალედური გამოცდის ქულა: ")
final_score = get_score("შეიტანეთ დასკვნითი გამოცდის ქულა: ")
project_score = get_score("შეიტანეთ პროექტის ქულა: ")

# გამოთვალეთ საშუალო ქულა
average_score = (midterm_score * 0.2) + (final_score * 0.4) + (project_score * 0.4)

# შედეგი
if average_score >= 70:
    print(f"გილოცავთ! თქვენ გაიარეთ კურსი. თქვენი საშუალო ქულა: {average_score:.2f}")
else:
    print(f"სამწუხაროდ, ვერ გაიარეთ კურსი. თქვენი საშუალო ქულა: {average_score:.2f}")

File: group 25/level 072/homework/task 3/homework3.js
let score = Number(prompt("შეიყვანეთ ქულა 0 დან 100 მდე"));
    
if (score >= 90 && score <= 100) {
    alert("შეფასება A");
} else if (score >= 80 && score < 90) {
    alert("შეფასება B");
} else if (score >= 70 && score < 80) {
    alert("შეფასება C");
} else if (score >= 60 && score < 70) {
    alert("შეფასება D");
} else {
    alert("შეფასება F");
}